src/core_init_atmosphere/mpas_init_atm_gwd.F

```diff
@@ -1163,9 +1370,10 @@ end function get_hgamma
    !> \details Computation following Lott and Miller (QJRMS 1997)
    !
    !-----------------------------------------------------------------------
-   real (kind=RKIND) function get_hsigma()
+   real (kind=RKIND) function get_hsigma(nx, ny)
 
       implicit none
+      integer, intent(in) :: nx, ny
```

Comment:
As a stylistic comment, we almost invariably leave a blank line after an `implicit none` statement before declaring subroutine arguments.

Reply:
Done. Also changed other parts to follow this style